app/Main.hs

```diff
@@ -43,15 +38,23 @@ import CardanoNode (
 -- Hydra auction CLI imports
 import CLI.Actions (CliAction, handleCliAction)
 import CLI.Parsers (
-  CliInput (..),
+  CliInput (InteractivePrompt, Watch),
+  PromptOptions (..),
   getCliInput,
   parseCliAction,
  )
+import CLI.Watch (
+  watchAuction,
+ )
 
 main :: IO ()
 main = do
-  MkCliInput {cliVerbosity, cliActor, cliNodeSocket, cliNetworkId} <- getCliInput
+  ci <- getCliInput
+  handleCliInput ci
 
+handleCliInput :: CliInput -> IO ()
+handleCliInput (Watch auctionName) = watchAuction auctionName
+handleCliInput (InteractivePrompt MkPromptOptions {..}) = do
```

Comment:
Do you mean because of `RecordWildCards`?

Reply:
yes, I don't like it with big records because I have to view interactively with hls to find out the type or grep around, especially with `(..)` imports

Reply:
Yeah, wildcards are not nice, agree.